File: Source/DroolTool.Models/DataTransferObjects/Announcement/AnnouncementDto.cs
using System;
using System.Collections.Generic;
using System.Text;
using DroolTool.Models.DataTransferObjects.User;

namespace DroolTool.Models.DataTransferObjects
{
    public class AnnouncementDto
    {
        public int AnnouncementID { get; set; }
        public DateTime AnnouncementDate { get; set; }
        public string AnnouncementTitle { get; set; }
        public string AnnouncementLink { get; set; }
        public string FileResourceGUIDAsString { get; set; }
        public UserSimpleDto LastUpdatedByUser { get; set; }
        public DateTime LastUpdatedDate { get; set; }

    }
}

File: Source/DroolTool.Web/src/app/shared/models/announcement/announcement-dto.ts
import { UserSimpleDto } from '../user/user-simple-dto';

export class AnnouncementDto{

    public AnnouncementID: number;
    public AnnouncementTitle:string;
    public AnnouncementLink?: string;
    public AnnouncementDate: Date;
    public LastUpdatedByUser:UserSimpleDto;
    public LastUpdatedDate: Date;
    public FileResourceGUIDAsString: string;

    constructor(obj?: any){
        Object.assign(this, obj);
    }
}
